1.0/docs/devguide/styling.md

```diff
@@ -699,3 +699,21 @@ that includes a `custom-style` element.
       } 
     </style>      
 
+## Third-party libraries that modify local DOM
+
+If you are using a third-party library that adds local DOM nodes to your
+Polymer element, you may notice that styles on the element do not update
+properly. 
+
+The correct way to add DOM nodes to a Polymer element's local DOM is via
+`Polymer.dom`. This method adds the proper scoping classes
+to the element, which ensures that styles are updated properly. 
+
+Many libaries do not know about this method, so they do not add DOM nodes 
```

Comment:
Instead of "correctly"—since there's a strong argument that the way third-party libraries are doing this is perfectly correct—might say, "Third-party libraries that are not designed to work with Polymer's local DOM implementation." Or "with the Polymer DOM API"